[IMP] product_multi_barcode: Performance

GabbasovDinar opened the pull request:
Improve search speed when searching products by name.

Task: 4183

The similar solution for a similar problem: https://github.com/OCA/stock-logistics-tracking/pull/40

GabbasovDinar committed:
[IMP] product_multi_barcode: Performance

Improve search speed when searching products by name.

Task: 4183

ivs-cetmix commented:
Hey @rousseldenis this is the same patch as we did for our module here: https://github.com/OCA/stock-logistics-tracking/pull/40
I think this should be a good performance improvement for this module as well.

ivs-cetmix reviewed (approved):
Code review LGTM.

Commit:
[IMP] product_multi_barcode: readme improved

Improve Readme USAGE, DESCRIPTIOM files and add the screenshot to be more user-friendly

Bearnard21 reviewed (approved):
Functional LGTM

Bearnard21 commented:
@jbaudoux please take a look

ivs-cetmix commented:
Hey @OCA/logistics-maintainers  let's make OCA faster together! 😄

rvalyi reviewed (approved):
LGTM

rvalyi commented:
/ocabot merge minor
